engine.py

```diff
@@ -297,6 +292,18 @@ def get_shotgun_info(self, mari_entity):
         """
         return self.__metadata_mgr.get_metadata(mari_entity)
 
+    def set_project_metadata(self, mari_project, context):
```

Comment:
projectmanager doesn't need to call this as `engine.create_project()` already does that. Should I leave this exposed or remove it?
Ideally, metadata should be set on project creation. I can't think of a reason we would want to switch metadata later (updating version is already handled. Otherwise, a project should stick with a single context...?)

Reply:
Yes, I can't think of a reason to keep this exposed either.

Reply:
Removed.